File: app/src/main/java/com/kevin/kggank/ui/view/IlistView.java
package com.kevin.kggank.ui.view;

import com.kevin.kggank.base.IBaseView;
import com.kevin.kggank.entity.GanhuoEntity;
import com.kevin.kggank.entity.GanhuoOneDayEntity;

import java.util.List;

/**
 * Created by kevin on 16/5/18.
 */
public interface IlistView extends IBaseView {

    public void getGanhuoListGril(List<GanhuoEntity> entities);
}

File: app/src/main/java/com/kevin/kggank/ui/view/IlistView.kt
package com.kevin.kggank.ui.view

import com.kevin.kggank.entity.GanhuoEntity
import com.kevin.kglib.base.LibIBaseView

/**
 * Created by kevin on 16/5/18.
 */
interface IlistView : LibIBaseView {

    fun getGanhuoListGril(entities: List<GanhuoEntity>)
}
